test/unit/lib/plugins/aws/package/compile/events/kafka.test.js

```diff
@@ -144,268 +149,406 @@ describe('test/unit/lib/plugins/aws/package/compile/events/kafka.test.js', () =>
       expect(eventSourceMappingResource.Properties).to.deep.equal(eventConfig.resource(awsNaming));
     };
 
-    it('should correctly compile EventSourceMapping resource properties for VPC_SECURITY_GROUP', async () => {
-      const eventConfig = {
-        event: {
-          topic,
-          bootstrapServers: ['abc.xyz:9092'],
-          accessConfigurations: { vpcSecurityGroup: 'sg-abc4567890' },
-        },
-        resource: (awsNaming) => {
-          return {
-            SelfManagedEventSource: {
-              Endpoints: {
-                KafkaBootstrapServers: ['abc.xyz:9092'],
+    describe.only('accessConfigurations', () => {
```

Comment:
I believe there's a leftover `only` left here, let's remove it

Reply:
good catch, thanks